src/endpoints/stats/count_minted_domains.rs

```diff
@@ -0,0 +1,118 @@
+use crate::{models::AppState, utils::get_error};
+use axum::{
+    extract::{Query, State},
+    http::{HeaderMap, HeaderValue, StatusCode},
+    response::IntoResponse,
+    Json,
+};
+use axum_auto_routes::route;
+use mongodb::bson::{doc, Bson};
+use serde::{Deserialize, Serialize};
+use std::sync::Arc;
+
+#[derive(Serialize)]
+pub struct CountMintedDomainsData {
+    count: u64,
+}
+
+#[derive(Deserialize)]
+pub struct CountDomainsQuery {
+    since: i64,
```

Comment:
Don't add a since parameter, as said in the issue, it should count all domains ever minted